src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,15 @@
+$day-size: 100px;
+$row-days: 7;
+$days-gap: 1px;
+$padings: 10px;
+
+$calendar-width: $row-days * $day-size + $padings * 2;
+
+$day-of-week:
+  "mon" 0,
+  "tue" 1 * ($day-size + $days-gap),
+  "wed" 2 * ($day-size + $days-gap),
+  "thu" 3 * ($day-size + $days-gap),
+  "fri" 4 * ($day-size + $days-gap),
+  "sat" 5 * ($day-size + $days-gap),
+  "sun" 6 * ($day-size + $days-gap);
```

Comment:
you can store only week day and number of day
and create cycle that will go through the days and set correct margins to needed selectors
so you shouldn't repeat this formula many times